django_rq/views.py

```diff
@@ -58,6 +59,28 @@ def stats(request):
     return render(request, 'django_rq/stats.html', context_data)
 
 
+def _collect_workers_per_configuration(queue_list):
+    """Collects, into a list, dictionaries of connections_config and its
+    workers
+    """
+    workers_collections = []
```

Comment:
Please change this to a dict instead, keyed by `connection_config` with each key having a list of workers as its value.